languages/python/oso/oso/oso.py

```diff
@@ -38,3 +38,24 @@ def is_allowed(self, actor, action, resource) -> bool:
             return True
         except StopIteration:
             return False
+
+    def get_allowed_actions(self, actor, resource, allow_wildcard=False):
+        # Get allowed actions on the resource
+        results = self.query_rule("allow", actor, Variable("action"), resource)
+        actions = set()
+        for result in results:
+            action = result.get("bindings").get("action")
+            if type(action) == Variable:
```

Comment:
Some Python linters/IDEs will complain that this should be `isinstance(action, Variable)`.

Reply:
Done.